index.js

```diff
@@ -76,21 +35,17 @@ function startServer(db) {
               err = { message: err || 'none' }
             }
             var statusCode = err.code || 500
-            var msg = {
-              op: 'request.summary',
-              code: statusCode,
-              route: req.route.name,
-              method: req.method,
-              path: req.url,
-              err: err,
-              t: Date.now() - req.time()
-            }
-            if (statusCode >= 500) {
-              log.error(msg)
-            }
-            else {
-              log.warn(msg)
-            }
+            api.emit(
+              'failure',
+              {
+                route: req.route.name,
+                method: req.method,
+                url: req.url,
+                err: err,
+                t: Date.now() - req.time()
```

Comment:
Ok, just fixed up logging a little bit. Here's two example messages for success/failure:

```
06:23:52.698Z  INFO db-api:  (op=request.summary, code=200, route=getaccountid, method=GET, path=/account/25AC26185D13CD236B5DA3529DC2D6E6, t=13)
06:23:58.489Z  WARN db-api:  (op=request.summary, code=404, route=getaccountid, method=GET, path=/account/25AC26185D13CD236B5DA3529DC2D6E, t=7)
    err: {
      "message": "Not Found",
      "errno": 116,
      "error": "Not Found",
      "code": 404
    }
```

Re-pushing in a couple of mins.